tests/validate_plusless_properties.py

```diff
@@ -16,7 +16,7 @@ def check_equivalency(key, node_type, node_properties):
         difference = diff(key_properties, counterpart_properties)
         raise Exception(f"{key} and {counterpart_key} both exist in {node_type}, but are different: {difference}")
 
-node_types = ['model_configs', 'seed_configs', 'snapshot_configs', 'source_configs', 'test_configs']
+node_types = ['model_configs', 'seed_configs', 'snapshot_configs', 'source_configs', 'test_configs', 'metric_configs']
```

Comment:
This was also alphabetical, which matters to no one but me (but does matter to me)

Reply:
PS: Good remembering that there was a new node type!